src/looprestoration.rs

```diff
@@ -726,3 +726,223 @@ pub(crate) unsafe extern "C" fn selfguided_filter<BD: BitDepth>(
         }
     };
 }
+
+// TODO(randomPoison): Temporarily pub until init logic is deduplicated.
+pub(crate) unsafe extern "C" fn sgr_5x5_c<BD: BitDepth>(
+    mut p: *mut libc::c_void,
+    stride: ptrdiff_t,
+    left: *const libc::c_void,
+    mut lpf: *const libc::c_void,
```

Comment:
```suggestion
    p: *mut libc::c_void,
    stride: ptrdiff_t,
    left: *const libc::c_void,
    lpf: *const libc::c_void,
```

Same unused `mut`s on args in the other `fn`s, too.

Reply:
Removed.